src/Instrument/ClassLoading/AopComposerLoader.php

```diff
@@ -72,9 +72,15 @@ public function __construct(ClassLoader $original, AspectContainer $container, a
         $prefixes     = $original->getPrefixes();
         $excludePaths = $options['excludePaths'];
 
-        // Let's exclude core dependencies from that list
-        $excludePaths[] = $prefixes['Dissect'][0];
-        $excludePaths[] = substr($prefixes['Doctrine\\Common\\Annotations\\'][0], 0, -16);
+        if (!empty($prefixes)) {
+            // Let's exclude core dependencies from that list
+            if (array_key_exists('Dissect', $prefixes)) {
```

Comment:
Could you please use `isset` instead of `array_key_exists()`? And also target 1.x for merge instead of master.